.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -7,6 +7,8 @@ body:
         ## Guidelines
         Please make sure you are running the latest version of Skript on a supported server platform and version.
         Try to make sure there are no issues of this same problem currently open either.
+        As of the release of Skript 2.10, the oldest supported version has been raised to 1.19.4.
+        Any issues created for versions older than 1.19.4 will not be looked into or fixed unless the issue persists on supported versions.
```

Comment:
Do you think it would be better to make this more generic rather than specifically about 2.10?

Reply:
Even if I make it more generic by taking out the relation to `2.10`, the oldest version to be supported should still be listed within the Guidelines. Which would then still need to be updated whenever the team decides to raise the bar. 
I personally believe it would be helpful to people who make issues to know why/when it changed.

But I can take it out, let me know.

Reply:
Oh yeah sure that's fine, I was meaning like saying more generally not to make issues for unsupported versions
(I think we might have some rule about what versions are supported? Is it the last three 1.X or so?)

Reply:
Hmmm, I see what you're saying, though it could potentially be misleading. For example, if I did put, "only the three latest major versions are supported", but then 1.22 comes out basically as soon as we release 2.10, it could make people think that 1.19.4 isnt supported anymore, when in reality that change wont take affect until 2.11.

That's how I'm perceiving it at least. Let me know

Reply:
Sure, it's not a change request anyway it's just a suggestion.